tests/integration.rs

```diff
@@ -68,7 +89,7 @@ fn test_exit_command() {
 #[test]
 fn test_cd_command() {
     let (stdout, stderr) = run_shell_command("cd /tmp && pwd");
-    assert_eq!(stdout, "/tmp");
+    assert_eq!(canonicalize_path(&stdout), canonicalize_path("/tmp"));
```

Comment:
Do we not only need the rhs to be canonicalized?

```rust
assert_eq!(stdout, canonicalize_path("/tmp"));
```

Reply:
Yeah you're right